images/daemon/cmd/daemon/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	kataDaemon "github.com/openshift/kata-operator-daemon/pkg/daemon"
+	kataTypes "github.com/openshift/kata-operator/api/v1"
+	mcfgapi "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	nodeapi "k8s.io/kubernetes/pkg/apis/node/v1beta1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func main() {
+
+	var kataOperation string
+	flag.StringVar(&kataOperation, "operation", "", "Specify kata operations. Valid options are 'install', 'upgrade', 'uninstall'")
+
+	var kataConfigResourceName string
+	flag.StringVar(&kataConfigResourceName, "resource", "", "Kata Config Custom Resource Name")
+	flag.Parse()
+
+	if kataOperation == "" {
+		fmt.Println("Operation type must be specified. Check -h for more information.")
+		os.Exit(1)
+	}
+	if kataConfigResourceName == "" {
+		fmt.Println("Kata Custom Resource name must be specified. Check -h for more information.")
+		os.Exit(1)
+	}
+
+	var kataActions kataDaemon.KataActions
+
+	kataClient, err := getKataConfigClient()
+	if err != nil {
+		fmt.Printf("Unable to get dynamic kata config client, %+v", err)
+		os.Exit(1)
+	}
+
+	kataActions = &kataDaemon.KataOpenShift{
+		KataClient: kataClient,
+		// KataBinaryInstaller: func() error {
```

Comment:
We can remove this commented out code. This piece of code is helpful in speeding up the debugging of operator and daemon logic by skipping the actual installation/uninstallation of the kata binaries.

Reply:
done